mapeditor/helper.cpp

```diff
@@ -23,7 +23,7 @@
 #include "../lib/mapping/MapFormatJson.h"
 #include "../lib/modding/ModIncompatibility.h"
 
-std::unique_ptr<CMap> Helper::openMapInternal(const QString & filenameSelect)
+std::unique_ptr<CMap> Helper::openMapInternal(const QString & filenameSelect, IGameInfoCallback * ecb)
```

Comment:
`ecb`? Editor callback? But you're passing interface, not editor implementation of it.
call it `cb` or `gameInfo` perhaps?